README.md

```diff
@@ -44,6 +44,15 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
+provider "restapi" {
+  uri = "https://private.resource-controller.cloud.ibm.com"
+  headers = {
+    Authorization  = xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX
```

Comment:
I think include the data lookup in the usage, and set this to
`data.ibm_iam_auth_token.auth_token.iam_access_token`